batch_flusher_test.go

```diff
@@ -0,0 +1,85 @@
+package iavl
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	dbm "github.com/cosmos/iavl/db"
+	"github.com/stretchr/testify/require"
+)
+
+type MockDBBatch struct {
+	dbm.Batch
```

Comment:
seems they removed `dbm.Batch` in https://github.com/cosmos/iavl/pull/955/files#diff-367b87ccd372ceca8c6f2e7157b827b4d469318a6571c0cacc99d54c1bb19089

Reply:
changed to corestore